plugins/woocommerce/src/Internal/FraudProtection/PaymentMethodEventTracker.php

```
@@ -0,0 +1,232 @@
+
+	/**
+	 * Handle payment method deleted event.
+	 *
+	 * Triggers fraud protection event tracking when a payment method is deleted.
+	 *
+	 * @internal
+	 *
+	 * @param int               $token_id The ID of the deleted token.
+	 * @param \WC_Payment_Token $token    The payment token object.
+	 */
+	public function handle_payment_method_deleted( $token_id, $token ): void {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$token_id'. (undefined)

(UnusedFormalParameter)